rpa-builder/6.0/modules/ROOT/pages/using-workflow-debugger.adoc

```diff
@@ -0,0 +1,57 @@
+= Using Workflow Debugger
+
+RPA Builder provides an integrated debugger for Workflows. To use it, click the *Debug Workflow* (image:debug-icon.png[The Debug Workflow icon, 2%, 2%]) or press _F8_. This starts a test run and opens Workflow Debugger instead of stopping the execution of the Workflow if an error occurs.
+
+When you execute a *Debug Workflow* run, you can call Workflow Debugger by pressing _Pause/Break_ on your keyboard. You can change the configuration of Workflow objects within the Workflow Debugger and then re-run them or continue from the current position. When the Workflow enters an error state, Workflow Debugger shows.
+
+image:workflow-debugger-window.png[The Workflow Debugger window]
+
+<1> The first pane in Workflow Debugger shows a condensed view of your Workflow and an arrow indicating which step was the last to execute.
+<2> On the second pane is the debug information for the selected Action Step. The Debug Information panel contains a combination of configuration settings and runtime information relating to Action Steps already executed.
+
+In this example, the Workflow entered the error state at the Window Wait Action Step. In the Debug Information panel, the *Result* section confirms that the window could not be found. Also, in the *Configuration* section, the *WindowTitle* value is set to `testname`, which causes the error because no window with that title exists during the execution.
+
+== Workflow Debugger Toolbar
+
+* (image:reload-workflow-icon.png[The Repeat icon, 2%, 2%]): *Repeat*
++
+Executes the Action Step again.
+* (image:edit-and-repeat-icon.png[The Edit & Repeat icon, 2%, 2%]): *Edit & Repeat*
++
+Opens the wizard for the last Action Step executed and enables you to modify its configuration and settings to fix the error. The Action Step then executes again to ensure it works correctly after the changes. The Workflow Debugger shows again on completion and you can check whether the *Run State* is _OK_ or _Error_.
+* (image:run-process.png[The Continue icon, 2%, 2%]): *Continue*
++
+Continues the Workflow execution.
++
+Note that the *Continue with Runstate:* drop-down menu reflects the current *Run State*. You can force the state to `Error` or `OK` at this point. For example, if you want to debug the _OnError_ part of a Managed Block, set the state to `Error` during the _DoAction_ section which subsequently leads to the execution of the _OnError_ section. You can also force the *Run State* to `OK` when you want the rest of the Workflow to be executed even though an error occurred.
```

Comment:
Just confirm the mix of italics and code here...I'm not sure what the italics are for _Do_